lib/Conversion/TorchToTcp/Utils.h

```diff
@@ -80,7 +80,10 @@ bool getConstTensorWithType(ConversionPatternRewriter &rewriter, Operation *op,
 template <typename TorchToTcpPattern, typename AtenOp>
 inline void addPatternIfOpInConvertTorchOpsSet(
     TypeConverter &typeConverter, RewritePatternSet &patterns,
-    ConversionTarget &target, const llvm::StringSet<> &convertTorchOpsSet) {
+    ConversionTarget &target, const llvm::StringSet<> &convertTorchOpsSet,
+    std::function<bool(AtenOp)> dynamicLegalityFcn = [](AtenOp) {
```

Comment:
Looks like the `dynamicLegalityFcn` is not being used below? I think you want this to be used to mark the op dynamically legal?